patches/amd-mainline/rocm-libraries/0025-Adding-gtest-target-link-for-rocFFT.patch

```diff
@@ -0,0 +1,93 @@
+From b394fd2138b97f394e3fd0209dc0e27f0f672556 Mon Sep 17 00:00:00 2001
+From: geomin12 <geomin12@amd.com>
+Date: Wed, 3 Sep 2025 08:45:05 -0700
+Subject: [PATCH 25/25] Adding gtest target link for rocFFT
+
+---
+ .../rocfft/clients/cmake/build-gtest.cmake    | 48 -------------------
+ projects/rocfft/clients/tests/CMakeLists.txt  | 14 +-----
+ 2 files changed, 2 insertions(+), 60 deletions(-)
+ delete mode 100644 projects/rocfft/clients/cmake/build-gtest.cmake
+
+diff --git a/projects/rocfft/clients/cmake/build-gtest.cmake b/projects/rocfft/clients/cmake/build-gtest.cmake
+deleted file mode 100644
+index 441eca65d3..0000000000
+--- a/projects/rocfft/clients/cmake/build-gtest.cmake
++++ /dev/null
```

Comment:
Let's delete `build-gtest.cmake` in rocfft (within rocm-libraries) directly, rather than via a patch.

Can match what the other subprojects do: https://github.com/search?q=repo%3AROCm%2Frocm-libraries%20find_package(gtest&type=code

Reply:
Actually there is no strict need to delete `build-gtest.cmake` at all if the `include` call is patched away. However, I would like to see a PR to upstream that applies the changes to `projects/rocfft/clients/tests/CMakeLists.txt`. I see two possibilities here:

1. Drop the custom finder and revise appropriately as done here.
2. Supported the old mechanism + find_package (or only the latter).

I vote for 1 but in case there is push back from upstream, we eventually can go with 2. However, I've sent this to other components before and this shouldn't be hard to land it upstream at all. We can carry a local copy of those patch but preferably this comes from a PR.